tests/runner/src/lib.rs

```diff
@@ -98,6 +101,7 @@ pub fn run_test_kernel_on_uefi_pxe(out_tftp_path: &Path) {
     run_qemu(args);
 }
 
+#[allow(dead_code)]
```

Comment:
We should use the proper `cfg` flags here instead of just silencing the dead code warning:

```suggestion
#[cfg(any(feature = "uefi", feature = "bios"))]
```